### ISSUE TITLE
rich text format support 

### ISSUE THREAD

**Issue by Reaper10** [olivierkes/manuskript#913]
Can/will you add  support for rich text format?

**Comment by TheJackiMonster** [comment]
Maybe it would be good to define required features for rich text because there can be many. I personally would like to have multiple kinds of alignment in one text file. However at the moment we're using mostly Markdown document which don't allow that. But since we technically use Multimarkdown to add custom headers, we could also include HTML inside of the Markdown text to implement rich text features.

I just think that this will need a lot of time from now. The focus is on stability and simplicity when it comes to files and structure. So rich text is pretty much an advancement which should be added once everything works properly for simpler formats.

**Comment by obw** [comment]
I think Markdown is the best possible format!

What Users which are not so technique orientated, could help from my side of view, is a WYSIWYG view in the editor. We have something like this at the moment, but the control characters are shown. Better for some users will be better, that only formatted Text should be shown.

Also, pictures should be displayed in this mode!

Regards

**Comment by AlanDunaway** [comment]
> I think Markdown is the best possible format!
> 
> What Users which are not so technique orientated, could help from my side of view, is a WYSIWYG view in the editor. We have something like this at the moment, but the control characters are shown. Better for some users will be better, that only formatted Text should be shown.
> 
> Also, pictures should be displayed in this mode!
> 
> Regards

Yeah, WYSIWYG editor is good when you want to read your project with all the text styles(italics, bold, underline) you apply. I love reading my own text in the editor.

**Comment by obw** [comment]
There is something with this concept, which would be absolut useful!

To add a custom CSS too it!

I have a Book, where all magic is written in red. I mark it while writing as bold, so when this feature comes, it would be perfect, when I con say per CSS, strong is not bold, but red!

If something like this is possible, per project, this would be a killer!

Also a rendering of footnotes, in a way, that they are at the bottom, would be a perfect addition!
